lib/tests/test_merged_tree.rs

```diff
@@ -181,6 +181,31 @@ fn test_from_legacy_tree() {
         merged_tree.value(&dir1_basename),
         MergedTreeValue::Resolved(tree.value(&dir1_basename))
     );
+
+    // Also test that MergedTreeBuilder can create the same tree by starting from an
+    // empty legacy tree.
+    let mut tree_builder = MergedTreeBuilder::new(
+        store.clone(),
+        MergedTreeId::Legacy(store.empty_tree_id().clone()),
+    );
+    for (path, value) in tree.entries_matching(&EverythingMatcher) {
```

Comment:
nit: `tree`/`merged_tree``.entries()` can be used here

Reply:
Done